### PR TITLE
incl./excl. options, path-from-stdin exclusiveness

### PR DESCRIPTION
**Pull request by elandorr**
Should we still bother with 1.2, or is v2 imminent?

### CODE REVIEW

File: src/borg/archiver.py [hunk 1]
```diff
@@ -3582,8 +3582,8 @@ def define_borg_mount(parser):
                                help='interpret PATH as command and store its stdout. See also section Reading from'
                                     ' stdin below.')
         subparser.add_argument('--paths-from-stdin', action='store_true',
-                               help='read DELIM-separated list of paths to backup from stdin. Will not '
-                                    'recurse into directories.')
+                               help='read DELIM-separated list of paths to backup from stdin. Exclusive option - no recursion'
+                                    ', no excludes, ``-x`` does not apply.')
```
Comment (ThomasWaldmann) [inline src/borg/archiver.py:3586]: Shall we just say "It will back up all files given, no more, no less."?

Comment (elandorr) [reply src/borg/archiver.py]: Sounds good to me. Is the v1 still relevant for more than a few weeks? No point bothering you with 1.2 PRs, if v2 is imminent.

Comment (ThomasWaldmann) [reply src/borg/archiver.py]: 1.2.x will be maintained for a longer time, most users have that in production, it is in stable dists and borg2 will still take a while (and even after it is released, people will need some time to adopt/migrate).

Comment (ThomasWaldmann) [reply src/borg/archiver.py]: #5248 (and there was no pressure).

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**